sources/web/datalab/static/util.js

```diff
@@ -1,7 +1,14 @@
 define(() => {
+
+  window.datalab = window.datalab || {};
+  window.datalab.debug = false;
```

Comment:
This seems reasonable. However for now we need something to hide the verbose logs we have on the client side.